mkosi/kmod.py

```diff
@@ -211,17 +212,25 @@ class ModuleDependencyInfo:
 
 
 def modinfo(context: Context, kver: str, modules: Sequence[str]) -> dict[str, ModuleDependencyInfo]:
-    cmdline = ["modinfo", "--modname", "--set-version", kver, "--null"]
+    cmdline = ["modinfo", "--set-version", kver, "--null"]
 
     if context.config.output_format.is_extension_image() and not context.config.overlay:
         cmdline += ["--basedir", "/buildroot"]
-        sandbox = context.sandbox(options=context.rootoptions(readonly=True))
+
+        def sandbox() -> AbstractContextManager[list[PathString]]:
+            return context.sandbox(options=context.rootoptions(readonly=True))
     else:
-        sandbox = chroot_cmd(root=context.rootoptions)
+
```

Comment:
Nit: the empty line here is a bit weird.

Reply:
ruff wants it